docs/compounds/post-card.html

```diff
@@ -81,27 +81,27 @@ <h3 class="mt-1 mb-1">5 things you’re doing wrong with your hair care routine<
 
             <h1>POST CARDS CAROUSEL</h1>
             <article class="post-card mb-4 mb-lg-3">
-                <figure id="carouselWithGroupedControls" class="carousel slide mb-2 no-gutters__in-container" data-ride="carousel">
+                <figure id="carouselWithGroupedControls" class="carousel carousel--fixed-height slide mb-2 no-gutters__in-container" data-ride="carousel">
                     <ol class="carousel-indicators">
                         <li data-target="#carouselWithGroupedControls" data-slide-to="0" class="active rounded-circle border-0"></li>
                         <li data-target="#carouselWithGroupedControls" data-slide-to="1" class="rounded-circle border-0"></li>
                         <li data-target="#carouselWithGroupedControls" data-slide-to="2" class="rounded-circle border-0"></li>
                     </ol>
                     <div class="carousel-inner">
-                        <div class="carousel-item active">
-                            <figure class="mb-0">
-                                <img src="https://via.placeholder.com/375x190.jpg/EFADBA" class="w-100">
-                            </figure>
-                        </div>
-                        <div class="carousel-item">
-                            <figure class="mb-0">
-                                <img src="https://via.placeholder.com/375x190.jpg/EFADBA" class="w-100">
-                            </figure>
-                        </div>
-                        <div class="carousel-item">
-                            <figure class="mb-0">
-                                <img src="https://via.placeholder.com/375x190.jpg/EFADBA" class="w-100">
-                            </figure>
+                      <div class="carousel-item text-center active">
```

Comment:
if we apply text-center to the parent all slides will get the property and we don't have to add the class for each slide

Reply:
actually I am not sure why I added text-center, it seems fine without it, I removed it